src/shared/lib/NumberUtils.ts

```diff
@@ -5,4 +5,12 @@ export function toConditionalPrecision(number: number, precision: number, thresh
     }
 
     return number.toPrecision(precision);
-};
\ No newline at end of file
+};
+
+/** will return false if number is null
+ * used to test FACETS data values which are imported into database as MIN_FLOAT when "NA"
+ */
+export function floatValueIsNA(number: number): boolean {
```

Comment:
The method should be tested

Reply:
@zhx828 also added a new utility function in MutationUtil. Where are tests setup in the frontend codebase?

Reply:
@averyniceday for unit tests, as long as the file name has extension .spec.ts it will be catched by the test suit. Checkout this file for example https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/studyView/StudyViewUtils.spec.ts

for e2e test, checkout this section  https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/studyView/StudyViewUtils.spec.ts